schematics/ng-add/rules/add-dependencies.ts

```diff
@@ -56,6 +58,15 @@ function getSingleSpaAngularDependency(): NodeDependency {
   };
 }
 
+function getStyleLoaderDependency(): NodeDependency {
+  return {
+    name: 'style-loader',
+    overwrite: false,
+    type: NodeDependencyType.Dev,
+    version: peerDependencies?.['style-loader'] || dependencies?.['style-loader'] || 'latest',
```

Comment:
This makes me a bit nervous because ideally we'd use the version of style-loader included in angular devkit or angular cli, rather than our own peer dependency. Was style-loader removed from those? Is there a way we can use it?